react/components/LoggedOutCTA/components/UI/index.js

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+
+import constants from 'react/styles/constants';
+import Text from 'react/components/UI/Text';
+import { GenericButtonLink } from 'react/components/UI/GenericButton';
+
+const Copy = styled(Text).attrs({
+  f: 5,
+  py: 3,
+  pr: 4,
+})``;
+
+const Button = styled(GenericButtonLink).attrs({
+  mx: 3,
+})`
+  ${constants.media.mobile`
+    padding-left: 0.5em;
+    padding-right: 0.5em;
+  `}
```

Comment:
This is like fine (use scale values though). But if you want you can also just configure this with an array where each value in the breakpoints https://github.com/aredotna/ervell/blob/master/react/styles/constants.js#L4-L5